bluepyemodel/emodel_pipeline/emodel_metadata.py

```diff
@@ -179,3 +183,23 @@ def as_string(self, seed=None, use_allen_notation=True):
             s += f"seed={seed}__"
 
         return s[:-2]
+
+    def check_name(self, emodel_name):
```

Comment:
Could you rename this to `check_emodel_name` or something similar, to be as explicit as possible?

Also, since you put this as a method and not a function external to the class, how about we check right after setting the `self.emodel_name`, and drop the (`emodel_name`) argument and use the class `self.emodel_name` attribute?